### DISCUSSION TITLE
Animate a user icon temporarily upon joining a voice channel to make it easier for existing members to identify the joined member.

### DISCUSSION THREAD

**Discussion by aalekhpatel07** [stoatchat/discussions#622] [Feature Suggestions]
Currently, when a user joins a voice channel, there is no indication of the identity of the joined member. The members of the channel would have to go through the list manually to identify who joined the call. This can become a lot of work in sufficiently large parties.

As a potential fix, could we highlight, or blink, or animate in some way, the acknowledgement of a member joining the call? This could be a temporary thing for, say, 3s to help the other members identify the joined member.

My apologies if there's another feature that already solves this problem, in which case, I'd love to learn about its usage.

Thanks

**Comment by Rexogamer** [comment]
We're currently rewriting the frontend, which will be followed by a rewrite of the voice server - once these are done, we will be able to look into improving the voice call experience ^^
